install

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-app="$( cd "$( dirname "$( readlink -f "${BASH_SOURCE[0]}" )" )" && pwd )"
+app=$( cd "$(dirname "$0")" ; pwd -P )
```

Comment:
ahh nice. I was working on something using Python, this doesn't resolve symlinks like it did before, but will get a clean path which is good enough for the app to work properly. 

Given it a few tests with all the scenarios I was using on the previous string - all good.